easybuild/easyconfigs/j/jax/jax-0.8.1-gcccoreflexiblas-14.3.0-3.4.5.eb

```
@@ -0,0 +1,108 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Denis Kristak
+# Updated by: Alex Domingo (Vrije Universiteit Brussel)
+# Updated by: Thomas Hoffmann (EMBL Heidelberg)
+# Updated by: Pavel Tománek (INUITS)
+
+easyblock = 'PythonBundle'
+
+name = 'jax'
+version = '0.8.1'
+
+homepage = 'https://jax.readthedocs.io/'
+description = """Composable transformations of Python+NumPy programs:
+differentiate, vectorize, JIT to GPU/TPU, and more"""
+
+toolchain = {'name': 'gcccoreflexiblas', 'version': '14.3.0-3.4.5'}
+
+builddependencies = [
+    ('Bazel', '7.4.1', '-Java-21'),
```

```suggestion
    ('Bazel', '7.7.0', '-Java-21'),
```

Should be 7.7.0 based on the [`.bazelversion`](https://github.com/jax-ml/jax/blob/jax-v0.8.1/.bazelversion)